### PR TITLE
Feature/GitHub actions

### PR DESCRIPTION
**Pull request by wmcginty**
There are a few tests that are failing because of entitlements issues in the test target. I figured they were best left for a different PR, unless someone who is more familiar with them wants to take a stab at fixing them now.

### CODE REVIEW

**Review by br-earl-gaspard — APPROVED** [review]

**Review by JobsIsMyHomeboy — APPROVED** [review]

File: Tests/KeychainStorageContainerTests.swift [hunk 1]
```diff
@@ -75,70 +75,70 @@ class KeychainStorageContainerTests: XCTestCase {
...
-            let alternateInstallKeychainContainer = KeychainStorageContainer<MockUser>(config: alternateInstallKeychainConfig)
-            
-            //Check that the data is no longer accessibile
-            let alternateStoredItem = try alternateInstallKeychainContainer.item(forIdentifier: identifier)
-            XCTAssertNil(alternateStoredItem)
-        } catch {
-            XCTFail("\(error)")
-        }
-    }
+//    func test_KeychainStorageContainer_HasItem() {
```
Comment (br-tyler-milner) [inline Tests/KeychainStorageContainerTests.swift:78]: Is the plan to reenable these tests eventually?

Comment (wmcginty) [reply Tests/KeychainStorageContainerTests.swift:78]: Yea, they hadn't been "working" for some time, but Travis wasn't running to catch it. Definitely plan to re-enable them at some point. I'll create an issue for it.

Comment (br-tyler-milner) [reply Tests/KeychainStorageContainerTests.swift:78]: Ah, good ol' Travis...good riddance!

Comment (wmcginty) [reply Tests/KeychainStorageContainerTests.swift:78]: Created #35 to track it

**Pull request merged**